duo-client/src/main/java/com/duosecurity/client/Http.java

```diff
@@ -36,6 +38,7 @@ public class Http {
   Map<String, String> params = new HashMap<String, String>();
   private Random random = new Random();
   private OkHttpClient httpClient;
+  private Map<String, String> additionalHeaders = new CaseInsensitiveMap<String, String>();
```

Comment:
what are the benefits of using the CaseInsensitiveMap?  in general we want to have as few dependencies as possible.